tst/EnergyPlus/unit/DXCoils.Unit.cc

```diff
@@ -0,0 +1,178 @@
+	DXCoil( DXCoilNum ).MSMaxONOFFCyclesperHour.allocate( DXCoil( DXCoilNum ).NumOfSpeeds );
+	DXCoil( DXCoilNum ).MSLatentCapacityTimeConstant.allocate( DXCoil( DXCoilNum ).NumOfSpeeds );
+	DXCoil( DXCoilNum ).MSFanPowerPerEvapAirFlowRate.allocate( DXCoil( DXCoilNum ).NumOfSpeeds );
+
+	DXCoil( 1 ).MSRatedTotCap( 1 ) = 4455.507579219055;
+	DXCoil( 1 ).MSRatedTotCap( 2 ) = 6188.507579219055;
+
+	DXCoil( DXCoilNum ).MSRatedTotCap( 1 ) = 4455.507579219055;
+	DXCoil( DXCoilNum ).MSRatedTotCap( 2 ) = 6188.204971137576;
+	DXCoil( DXCoilNum ).MSRatedCOP( 1 ) = 4.03;
+	DXCoil( DXCoilNum ).MSRatedCOP( 2 ) = 3.53;
+
+	DXCoil( DXCoilNum ).MSCCapFFlow = 1;
+	DXCoil( DXCoilNum ).MSCCapFTemp = 3;
+	DXCoil( DXCoilNum ).MSEIRFFlow = 1;
+	DXCoil( DXCoilNum ).MSEIRFTemp = 3;
+	DXCoil( DXCoilNum ).MSPLFFPLR = 2;
+	DXCoil( DXCoilNum ).MSRatedAirVolFlowRate( 1 ) = 0.2339;
+	DXCoil( DXCoilNum ).MSRatedAirVolFlowRate( 2 ) = 0.2924;
+	DXCoil( DXCoilNum ).MSFanPowerPerEvapAirFlowRate = 0.0;
+	DXCoil( DXCoilNum ).RegionNum = 4;
+	DXCoil( DXCoilNum ).MinOATCompressor = -17.78;
+
+
+	NumCurves = 3;
+	PerfCurve.allocate( NumCurves );
+
+	CurveNum = 1;
+	PerfCurve( CurveNum ).CurveType = Quadratic;
+	PerfCurve( CurveNum ).ObjectType = CurveType_Quadratic;
+	PerfCurve( CurveNum ).InterpolationType = EvaluateCurveToLimits;
+	PerfCurve( CurveNum ).Coeff1 = 1;
+	PerfCurve( CurveNum ).Coeff2 = 0.0;
+	PerfCurve( CurveNum ).Coeff3 = 0.0;
+	PerfCurve( CurveNum ).Coeff4 = 0.0;
+	PerfCurve( CurveNum ).Coeff5 = 0.0;
+	PerfCurve( CurveNum ).Coeff6 = 0.0;
+	PerfCurve( CurveNum ).Var1Min = 0.0;
+	PerfCurve( CurveNum ).Var1Max = 2.0;
+	PerfCurve( CurveNum ).Var2Min = 0.0;
+	PerfCurve( CurveNum ).Var2Max = 2.0;
+
+	CurveNum = 2;
+	PerfCurve( CurveNum ).CurveType = Quadratic;
+	PerfCurve( CurveNum ).ObjectType = CurveType_Quadratic;
+	PerfCurve( CurveNum ).InterpolationType = EvaluateCurveToLimits;
+	PerfCurve( CurveNum ).Coeff1 = 1;
+	PerfCurve( CurveNum ).Coeff2 = 0.0;
+	PerfCurve( CurveNum ).Coeff3 = 0.0;
+	PerfCurve( CurveNum ).Coeff4 = 0.0;
+	PerfCurve( CurveNum ).Coeff5 = 0.0;
+	PerfCurve( CurveNum ).Coeff6 = 0.0;
+	PerfCurve( CurveNum ).Var1Min = 0.0;
+	PerfCurve( CurveNum ).Var1Max = 1.0;
+	PerfCurve( CurveNum ).Var2Min = 0.7;
+	PerfCurve( CurveNum ).Var2Max = 1.0;
+
+	CurveNum = 3;
+	PerfCurve( CurveNum ).CurveType = BiQuadratic;
+	PerfCurve( CurveNum ).ObjectType = CurveType_BiQuadratic;
+	PerfCurve( CurveNum ).InterpolationType = EvaluateCurveToLimits;
+	PerfCurve( CurveNum ).Coeff1 = 1;
+	PerfCurve( CurveNum ).Coeff2 = 0.0;
+	PerfCurve( CurveNum ).Coeff3 = 0.0;
+	PerfCurve( CurveNum ).Coeff4 = 0.0;
+	PerfCurve( CurveNum ).Coeff5 = 0.0;
+	PerfCurve( CurveNum ).Coeff6 = 0.0;
+	PerfCurve( CurveNum ).Var1Min = -100.0;
+	PerfCurve( CurveNum ).Var1Max = 100.0;
+	PerfCurve( CurveNum ).Var2Min = -100.0;
+	PerfCurve( CurveNum ).Var2Max = 100.0;
+
+	SetPredefinedTables();
+	SizeDXCoil( 2 );
+	EXPECT_EQ( 5000.0, DXCoil( 2 ).DefrostCapacity );
```

Comment:
@nmerket I don't have a problem to make a change.
@Myoldmopar Do you agree it?

In addition, I only changed a line of code. How to make tests failed? Any idea?